presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergMetadataFactory.java

```diff
@@ -56,7 +56,8 @@ public ConnectorMetadata create()
     {
         switch (catalogType) {
             case HADOOP:
-                return new IcebergHadoopMetadata(resourceFactory, typeManager, commitTaskCodec);
+            case NESSIE:
```

Comment:
Nit: to be more consistent, we could put all the cases in alphabetical order

Reply:
that's what I had initially, but Intellij would then complain due to duplicate case branches. So I figured that it's better to merge `HADOOP` + `NESSIE` together

Reply:
@beinan let me know if you would like me to still change this and introduce duplicate case branches